src/util/MutationDataUtils.ts

```diff
@@ -163,70 +85,63 @@ export function calculateTotalBiallelicRatio(mutations: IExtendedMutation[],
     return ratio;
 }
 
-function combinedBiallelicCountsByTumorType(mutations: IExtendedMutation[], cancerTypeFilter?: CancerTypeFilter)
+function combinedBiallelicCountsByTumorType(mutations: IExtendedSignalMutation[], cancerTypeFilter?: CancerTypeFilter)
 {
     return combinedCounts(mutations,
-        (mutation: IExtendedMutation) => mutation.biallelicCountsByTumorType,
+        (mutation: IExtendedSignalMutation) => mutation.biallelicCountsByTumorType,
         cancerTypeFilter);
 }
 
-function combinedQcPassCountsByTumorType(mutations: IExtendedMutation[], cancerTypeFilter?: CancerTypeFilter)
+function combinedQcPassCountsByTumorType(mutations: IExtendedSignalMutation[], cancerTypeFilter?: CancerTypeFilter)
 {
     return combinedCounts(mutations,
-        (mutation: IExtendedMutation) => mutation.qcPassCountsByTumorType,
+        (mutation: IExtendedSignalMutation) => mutation.qcPassCountsByTumorType,
         cancerTypeFilter);
 }
 
-function combinedTumorTypeDecompositions(mutations: IExtendedMutation[], cancerTypeFilter?: CancerTypeFilter)
+function combinedTumorTypeDecompositions(mutations: IExtendedSignalMutation[], cancerTypeFilter?: CancerTypeFilter)
 {
     return combinedCounts(mutations,
-        (mutation: IExtendedMutation) => mutation.tumorTypeDecomposition,
+        (mutation: IExtendedSignalMutation) => mutation.tumorTypeDecomposition,
         cancerTypeFilter);
 }
 
-function combinedCounts(mutations: IExtendedMutation[],
-                        getCounts: (mutation: IExtendedMutation) => ICountByTumorType[] | undefined,
+function combinedCounts(mutations: IExtendedSignalMutation[],
+                        getCounts: (mutation: IExtendedSignalMutation) => CountByTumorType[] | undefined,
                         cancerTypeFilter?: CancerTypeFilter)
 {
     return _.flatten(mutations.map(mutation =>
         filterCountsByTumorType(getCounts(mutation), cancerTypeFilter)));
 }
 
-function filterCountsByTumorType(counts?: ICountByTumorType[], cancerTypeFilter?: CancerTypeFilter)
+function filterCountsByTumorType(counts?: CountByTumorType[], cancerTypeFilter?: CancerTypeFilter)
 {
     return counts ? counts.filter(c => containsCancerType(cancerTypeFilter, c.tumorType)) : [];
 }
 
-function totalVariants(counts: ICountByTumorType[]) {
+// TODO duplicate of https://bit.ly/3qQ13V5
```

Comment:
We have `totalVariants` and `totalSamples` in `cbioportal-utils`, but they are not exported. We should export and replace these two.

Reply:
I guess we can do this in the next package update.